tests/utils/managed_process.py

```
@@ -0,0 +1,242 @@
+
+        for ps_process in psutil.process_iter(["name", "cmdline"]):
+            if ps_process.name() in self.stragglers:
+                self._terminate_process_tree(ps_process.pid)
+
+    def _start_process(self):
+        assert self._command_name
+        assert self._log_path
+
+        self._logger.info(
+            f"Running command: {' '.join(self.command)} in {self.working_dir or os.getcwd()}"
+        )
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)